go-packages/meep-sandbox-ctrl-client/api_applications.go

```diff
@@ -298,18 +298,16 @@ This method retrieves information about a list of mec application resources.
  * @param ctx context.Context - for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
  * @param optional nil or *ApplicationsGETOpts - Optional Parameters:
      * @param "App" (optional.String) -  Filter by application name
-     * @param "State" (optional.String) -  Filter by application state
-     * @param "Type_" (optional.String) -  Filter by application type
      * @param "Mep" (optional.String) -  Filter by MEP name
```

Comment:
refer to node instead of mep
Applies to the rest of this module

Reply:
Fixed in PR #327